### PR TITLE
cli: Show node identities in acl token list output

### PR DESCRIPTION
**Pull request by blake**
Fix pretty CLI output of `consul acl token list` so that it properly displays node identities that are associated with a token.

### CODE REVIEW

File: command/acl/token/formatter.go [hunk 1]
```diff
@@ -173,13 +173,9 @@ func (f *prettyFormatter) formatTokenListEntry(token *api.ACLTokenListEntry) str
...
-		buffer.WriteString(fmt.Sprintln("Service Identities:"))
-		for _, svcid := range token.ServiceIdentities {
-			if len(svcid.Datacenters) > 0 {
-				buffer.WriteString(fmt.Sprintf("   %s (Datacenters: %s)\n", svcid.ServiceName, strings.Join(svcid.Datacenters, ", ")))
-			} else {
-				buffer.WriteString(fmt.Sprintf("   %s (Datacenters: all)\n", svcid.ServiceName))
-			}
+		buffer.WriteString(fmt.Sprintln("Node Identities:"))
+		for _, nodeid := range token.NodeIdentities {
+			buffer.WriteString(fmt.Sprintf("   %s (Datacenter: %s)\n", nodeid.NodeName, nodeid.Datacenter))
```
Comment (FFMMM) [inline command/acl/token/formatter.go:178]: i may be wrong here but I think this is breaking some of the units 🤔 

re-running the current failed ones.

Comment (blake) [reply command/acl/token/formatter.go:178]: The failing tests seem to be related to anti-entropy sync, whereas this PR only changes the output of one of the CLI commands. I'm also able to reproduce the test failures on the `main` branch, so I suspect that they're unrelated to these changes.

**Review by karl-cardenas-coding — APPROVED** [review]
👍🏼  from a docs perspective

File: .changelog/11926.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
```
Comment (dnephin) [inline .changelog/11926.txt:1]: minor: seems like a bug fix since we were printing the wrong values before

**Review by dnephin — APPROVED** [review]
LGTM, thanks for fixing this!

File: .changelog/11926.txt (hunk 2 above)
Comment (blake) [reply .changelog/11926.txt]: Done. I've changed this to `bug` in the latest commit.

**Review by dnephin — APPROVED** [review]

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/539800.

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/539839.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit e898cf1d416a50599706054a949c2c16b0276dca onto `release/1.11.x` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit e898cf1d416a50599706054a949c2c16b0276dca onto `release/1.10.x` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit e898cf1d416a50599706054a949c2c16b0276dca onto `release/1.9.x` succeeded!

**Pull request merged**